apps/ledger-live-mobile/android/app/src/main/java/com/ledger/live/BackgroundRunner.kt

```diff
@@ -0,0 +1,90 @@
+package com.ledger.live;
+
+import android.app.PendingIntent
+import android.content.Intent
+import androidx.core.app.NotificationCompat
+import androidx.core.app.NotificationManagerCompat
+import com.facebook.react.HeadlessJsTaskService
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.ReactContextBaseJavaModule
+import com.facebook.react.bridge.ReactMethod
+
+
+/**
+ * This class is in charge of receiving the call from react-native side and in turn calling a
+ * headlessJS service that will run our code in the background.
+ */
+class BackgroundRunner(var context: ReactApplicationContext) : ReactContextBaseJavaModule(context) {
```

Comment:
the changes is quite important and we may want to involve another code reviewer as I don't know Kotlin well.

That said, the background runner addition, which sounds like a great feature, doesn"t seem presented & explained in the PR description and would worth to have some documentation attached.

Reply:
@juan-cortes had reviewed it while it was on the llm repo

I'll add some info on the PR about the background runner. But in short, for app installs we have some dependency on the backend (BIM)

Reply:
@gre this was the background installation in the background on HeadlessJS, only works on Android, hence the need to have the whole BIM paradigm. It _was_ presented six months ago :trollface: